### DISCUSSION TITLE
Copilot ⨯ Zed

### DISCUSSION THREAD

**Discussion by mikayla-maki** [community/community#50285] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi everyone, I'm going to be building Zed's copilot integration over the next few weeks. Looking forward to getting to know y'all as it comes together :D

**Comment by joeldrapper** [comment]
Oh wow, this is very exciting! 🎉

**Reply by richardhenry** [reply]
Super exciting!

**Comment by cktang88** [comment]
Hey @mikayla-maki is there any update you can provide on progress? I'm really excited about Zed's copilot integration!

**Comment by joeldrapper** [comment]
@mikayla-maki I just got the update. Can't believe how fast you got this working. 👏👏👏
